tutorials/quantize.md

```diff
@@ -24,6 +24,7 @@ To reduce the memory requirements further, Lit-GPT supports several quantization
 Enabled with [bitsandbytes](https://github.com/TimDettmers/bitsandbytes). Check out the [paper](https://arxiv.org/abs/2305.14314v1) to learn more about how it works.
 
 > **Note**: `bitsandbytes` only supports `CUDA` devices and the `Linux` operating system.
+Windows users should use [WSL2](https://learn.microsoft.com/en-us/windows/ai/directml/gpu-cuda-in-wsl).
```

Comment:
It looks like one of the users with a Windows machine [managed](https://github.com/Lightning-AI/lit-gpt/issues/278#issuecomment-1645520255) to run bnb inside WSL2 (which is basically a Linux in a VM).